sentinel/src/schedule/index.js

```diff
@@ -35,45 +39,32 @@ exports.sendable = function(config, now) {
   return now >= after && now <= until;
 };
 
+const init = () => {
+  Object.keys(tasks).forEach(taskName => {
+    if (ongoingTasks.has(taskName)) {
+      logger.debug(`Skipping Task ${taskName} as it's still running`);
```

Comment:
Consider making this `info` level otherwise it may be really hard to work out why a task isn't running...

Reply:
Hmmmmmmmmmmmm

Yeah OK. I was worried about "everything's OK alarm" esque spam, but at worst it's every 5min right now.

Changed.